<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>REPropertiesDialog</class>
 <widget class="QDialog" name="REPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>411</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>611</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tracksTab">
    <attribute name="title">
     <string>Tracks</string>
    </attribute>
    <widget class="QPushButton" name="deleteTrackButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>340</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="capoSpinBox">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>310</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="tuningButton">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>260</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QListWidget" name="midiProgramList">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>100</y>
       <width>181</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="nameLabel">
     <property name="geometry">
      <rect>
       <x>281</x>
       <y>40</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="capoLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>310</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Capo</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="tuningLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>260</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Tuning</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="mainTrackLabel">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Track</string>
     </property>
    </widget>
    <widget class="QLabel" name="stringCountLabel">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>288</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>(6 strings)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="midiProgramLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>100</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>MIDI Program</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="addTrackButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="shortNameLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>70</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Short Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="shortNameText">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>70</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameText">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tuningText">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>260</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="RETrackListView" name="trackList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="partsTab">
    <attribute name="title">
     <string>Parts</string>
    </attribute>
    <widget class="REPartListView" name="partList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="deletePartButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>340</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addPartButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="showStandardNotation">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>120</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Standard Notation</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="showTablature">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>150</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Tablature</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="hideDynamics">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>210</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Hide Dynamics</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="useMultibarRests">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>180</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Use Multibar Rests</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Part Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="partNameText">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Presentation Options</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RETrackListView</class>
   <extends>QListView</extends>
   <header>RETrackListView.h</header>
  </customwidget>
  <customwidget>
   <class>REPartListView</class>
   <extends>QListView</extends>
   <header>REPartListView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>REPropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>REPropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
